package.json

```diff
@@ -5,27 +5,46 @@
   "type": "module",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "prettier": "prettier --write '{src,cypress,test-data}/**/*.{ts,js,json,scss,html}' '*.{js,json}'",
-    "lint": "eslint --ext .js,.ts src/app --fix",
+    "start-metis": "ng serve metis",
+    "start": "npm run start-metis",
+    "build-metis": "ng build metis",
+    "build": "npm run build-metis",
+    "prettier-metis": "prettier --write '{projects/metis/src,projects/metis/cypress,projects/metis/test-data}/**/*.{ts,js,json,scss,html}' '*.{js,json}'",
+    "prettier": "npm run prettier-metis",
```

Comment:
If there is a second project, would it also be triggered if you run prettier? Or would you always have to make multiple calls to make sure you run prettier on everything? I think there should be a call for all projects (if you work on multiple), but this is perhaps not always possible (e.g. running start). What's the idea here?

Reply:
I'll address that in the follow up ticket once another project has been added.  Prettier doesn't take long to run, so the default action probably should be to run it on everything.  As configured at present it will only operate on the files under `projects/metis`

Reply:
OK, let's just keep that in mind.